feat(next13-zustand-bulma): add storybook

BigAB opened the pull request:
## Type of change

Add Storybook.

## Summary of change

I just ran `npx storybook@next init`, the locked the versions in `package.json` and committed.
 
## Checklist

<!-- Delete as appropriate and then go through the list, adding an X to every item you have completed -->

- [x] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [x] Changes for this new starter kit are being pushed to an integration branch instead of main
- [x] All dependencies are version locked
- [x] This fix resolves #993 
- [x] I have verified the fix works and introduces no further errors

jdwilkin4 reviewed (commented):
This looks good.

I think we should get rid of the default stories folder and assets since that won't be included in the final kit. 

Also, what are your thoughts on add the [accessibility addon](https://storybook.js.org/addons/@storybook/addon-a11y) to this kit?
I added it for the next-chakra ui kit and really like it

BigAB commented:
> I think we should get rid of the default stories folder and assets since that won't be included in the final kit.

Done in [6193c25](https://github.com/thisdot/starter.dev/pull/1183/commits/6193c2573ac2e5dc8b51fb226a5165f2a813effe)

> Also, what are your thoughts on adding the [accessibility addon](https://storybook.js.org/addons/@storybook/addon-a11y) to this kit? I added it for the next-chakra ui kit and really like it

Sounds great, added in [6193c25](https://github.com/thisdot/starter.dev/pull/1183/commits/6193c2573ac2e5dc8b51fb226a5165f2a813effe)

jdwilkin4 reviewed (approved):
looks good 👍
